web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -110,6 +110,8 @@ export type NavigationSubsection = {
    * Note that this is merely extra logic, and does not replace the default routing behaviour of a subsection which will navigate the user to the route.
    */
   onClick?: () => void;
+  /** isHyperlink is whether this subsection is merely a hyperlink/shortcut to another subsection. */
```

Comment:
We're sometimes using Hyperlink and other times using HyperLink (capital L). Let's be consistent.